Bottom: 701841d0a32b0299301f223974a38560ee614ae2
Top:    5dd56f3e495be66660e9b2c0ba41dd83faee5a23
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-13 08:00:16 +0800

Refresh of vhost_net-batch-update-used

---

diff --git a/virt/kvm/kvm_main.c b/virt/kvm/kvm_main.c
index 3ca33b6..c7b2e92 100644
--- a/virt/kvm/kvm_main.c
+++ b/virt/kvm/kvm_main.c
@@ -2142,7 +2142,7 @@ void kvm_vcpu_block(struct kvm_vcpu *vcpu)
 			 * This sets KVM_REQ_UNHALT if an interrupt
 			 * arrives.
 			 */
-			if (kvm_vcpu_check_bloxck(vcpu) < 0) {
+			if (kvm_vcpu_check_block(vcpu) < 0) {
 				++vcpu->stat.halt_successful_poll;
 				if (!vcpu_valid_wakeup(vcpu))
 					++vcpu->stat.halt_poll_invalid;
